app/models/manageiq/providers/openstack/infra_manager.rb

```diff
@@ -27,6 +27,7 @@ class ManageIQ::Providers::Openstack::InfraManager < ManageIQ::Providers::InfraM
   supports :events do
     unsupported_reason_add(:events, _("Events are not supported")) unless capabilities["events"]
   end
+  not_supported :shutdown
```

Comment:
@kbrock does this depend on changes in core?  Or was this supposed to be `supports_not :shutdown` ?

Reply:
it was a flub. thanks